src/openkit-dotnetpcl-4.5/MyClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
is this class needed or will it be extended later

Reply:
Nope, leftover when generating a new project.

Reply:
File has been removed.